libs/generate-ui/feature-task-execution-form/src/lib/task-execution-form/task-execution-form.component.ts

```diff
@@ -550,6 +597,43 @@ export class TaskExecutionFormComponent implements OnInit {
       ).join(' ')}`
     );
   }
+
+  hasFilteredOtherFields(
+    filteredFields: Set<string>,
+    otherFields: Set<string>
+  ): boolean {
+    return Array.from(filteredFields.values()).some((filtered) =>
+      otherFields.has(filtered)
+    );
+  }
+
+  hasFilteredImportantFields(
+    filteredFields: Set<string>,
+    important: Set<string>
+  ): boolean {
+    return Array.from(filteredFields.values()).some((filtered) =>
+      important.has(filtered)
+    );
+  }
+
+  toggleShowFields() {
+    this.showOtherFields = !this.showOtherFields;
+    this.changeDetectorRef.detectChanges();
+  }
+
+  getDocsLink(taskExecForm: TaskExecutionForm): string | undefined {
+    if (
+      !taskExecForm.architect.collection ||
+      !taskExecForm.architect.collection.includes('@nrwl')
+    ) {
+      return undefined;
+    }
+    const collectionStripped = taskExecForm.architect.collection.replace(
+      '@nrwl/',
```

Comment:
just an FYI - we might have to change this relatively soon to replace `@nx` as well